src/explore-education-statistics-admin/src/pages/release/pre-release/PreReleaseTableToolPage.tsx

```diff
@@ -41,30 +93,54 @@ const PreReleaseTableToolPage = ({
         locations: {},
       },
     };
-  }, [releaseId]);
+  }, [publicationId, releaseId, dataBlockId]);
 
   return (
-    <LoadingSpinner loading={isLoading}>
-      {tableToolState && (
-        <>
-          <PageTitle
-            title="Create your own tables online"
-            caption="Table Tool"
-          />
+    <>
+      <PageTitle title="Create your own tables online" caption="Table Tool" />
 
-          <p>
-            Choose the data and area of interest you want to explore and then
-            use filters to create your table.
-          </p>
+      <p>
+        Choose the data and area of interest you want to explore and then use
+        filters to create your table.
+      </p>
 
-          <p>
-            Once you've created your table, you can download the data it
-            contains for your own offline analysis.
-          </p>
+      <p>
+        Once you've created your table, you can download the data it contains
+        for your own offline analysis.
+      </p>
 
+      <LoadingSpinner loading={isLoading}>
+        {tableToolState && (
           <TableToolWizard
+            scrollOnMount
             themeMeta={[]}
             initialState={tableToolState}
+            renderHighlights={highlights => (
```

Comment:
Is there a reason to pass this as a render function here rather than pass the data and generate the JSX later? Maybe pull this out into a table highlights component to save duplication with TableToolPage?

Reply:
There's actually not a lot of commonality with the public `TableToolPage` as the `Link` components differ for the public vs admin. Having it abstracted into its own component would most likely need its _own_ render function :wink: